NetKVM/Common/ParaNdis-TX.h

```diff
@@ -3,6 +3,7 @@
 #include "ParaNdis-AbstractPath.h"
 #include "ParaNdis_GuestAnnounce.h"
 #include "ParaNdis_LockFreeQueue.h"
+#include "ParaNdis_DebugHistory.h"
```

Comment:
Please submit this as a separate PR. It can already be merged and make future review easier. 

Thanks!